packages/beacon-node/src/chain/opPools/opPool.ts

```diff
@@ -66,7 +75,7 @@ export class OpPool {
     ]);
 
     for (const attesterSlashing of attesterSlashings) {
-      this.insertAttesterSlashing(attesterSlashing.value, attesterSlashing.key);
+      this.insertAttesterSlashing(ForkName.electra, attesterSlashing.value, attesterSlashing.key);
```

Comment:
same as insert from api, we can just use electra type as it works for both phase0 and electra attester slashing